Teams_App/src/Components/SearchBarComponent/SearchBarComponent.js

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react";
+import "./SearchBarComponent.css";
+import CARDS_LIST from "../../Data/TeamData";
+
+const SearchBarComponent = ({ setCardsList }) => {
+  /**
+   * getFilteredData
+   * @param {*} search
+   * @returns List of filtered cards matching the searched pattern
+   */
+  const getFilteredData = (search) => {
+    return CARDS_LIST.filter((card) =>
```

Comment:
Make this function pure and reusable if you can.

Reply:
Yes I'll fix this up.